src/main/java/com/epam/edumanagementsystem/admin/journal/rest/service/impl/JournalServiceImpl.java

```diff
@@ -56,7 +57,9 @@ public void openJournal(String date, String startDate, String name, Model model)
         LocalDate timetableStartDate = timetableService.findTimetableByAcademicClassName(name).getStartDate();
         LocalDate timetableEndDate = timetableService.findTimetableByAcademicClassName(name).getEndDate();
         model.addAttribute(START_DATE, timetableStartDate);
+        model.addAttribute("statDateForDatePicker", timetableStartDate);
```

Comment:
What is the meaning to add the same value for both START_DATE and "statDateForDatePicker"